source/compute/gpu-support.rst

```diff
@@ -0,0 +1,235 @@
+.. _gpu-support:
+
+##############################
+GPU Support in Virtual Servers
+##############################
+
+**********************
+c2-gpu Virtual Servers
+**********************
+
+Each c2-gpu instance type includes one or more slices of NVIDIA A100
+GPUs. The slice size provided is "GRID A100D-20C", which provides
+2 compute pipelines and 20GB of video RAM from the card.
+
+.. warning::
+
+    c2-gpu virtual servers are in Technical Preview only
+
+Minimum Requirements
+====================
+
+For "c2-gpu", the absolute minimum requirements are as follows:
+
+* A boot/OS disk of at least 30GB (when installing CUDA support)
+* NVIDIA vGPU driver from the v15.0 series. This is currently version
```

Comment:
I'm wary about the wording of "This is currently version...". Perhaps "At the time of writing, this is version..."

Reply:
It has to be in lock-step, so the metal driver changes *must* update the documentation and the links to the current versions as well (since customers will need to use those exact links).

Reply:
🤷 the wording just sounds like we'd update it for every point release.